DoD.md

```diff
@@ -0,0 +1,19 @@
+# Definition of Done
+
+## A deliverable is to be considered “done” when
+
+1. The feature/bug fix implemented - code is complete, documented and checked in
+1. Pull request/commit includes bug number or JIRA number
+1. Unit tests written and running cleanly in the CI environment [1]
+1. All linters are running cleanly in the CI environment
+1. Failing CI blocks release/merge
+1. Integration tests written and running cleanly in the CI environment
+1. Code changes reviewed by at least two peers
+1. Acceptance criteria are verified and fulfilled
+1. Code merged in the main branch and running in Prod-preview
+1. User docs completed, reviewed and published (for user-facing changes)
+1. Code deployed in Production [2]
```

Comment:
all the number on this numbered list are 1, Will it work?

Reply:
yes, this is how to use ordered list in markdown (w/o need to edit when one item is added/deleted in the middle of list)